rl_runner.py

```diff
@@ -44,6 +44,10 @@ def loop(logdir=None):
             subprocess.call(
                 ("python rl_loop.py train --logdir=%s" % logdir).split())
 
+        with timer("validate"):
```

Comment:
Hm. Do we want the trainer to do both validation and training?

Reply:
yeah, why not?  validate is ~a minute, or about ~1% of overhead

Reply:
SGTM.

Reply:
We might re-visit this when we containerize the trainer; Brian mentioned that he'd like more validation steps, which would move us from 1% overhead up to 2% or 3% which seems like enough to break validation off of the main trainer process.